proms-resources/frontend/src/main/frontend/webpack.config.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
     [module_name + 'ToUConfiguration']: './src/proms/ToUConfiguration.jsx',
     [module_name + 'ToUConfigurationIcon']: '@mui/icons-material/Handshake.js',
     [module_name + 'SurveyInstrConfiguration']: './src/proms/SurveyInstrConfiguration.jsx',
-    [module_name + 'SurveyInstrConfigurationIcon']: '@mui/icons-material/Quiz.js'
+    [module_name + 'SurveyInstrConfigurationIcon']: '@mui/icons-material/Quiz.js',
+	[module_name + 'DashboardSettingsConfiguration']: './src/proms/DashboardSettingsConfiguration.jsx',
```

Comment:
Indentation artifact
```suggestion
    [module_name + 'DashboardSettingsConfiguration']: './src/proms/DashboardSettingsConfiguration.jsx',
```